java/vector/src/main/codegen/templates/UnionFixedSizeListWriter.java

```diff
@@ -197,7 +206,7 @@ public void writeDecimal(BigDecimal value) {
   @Override
   public void write${name}(<#list fields as field>${field.type} ${field.name}<#if field_has_next>, </#if></#list>) {
     if (writer.idx() >= (idx() + 1) * listSize) {
-      throw new IllegalStateException(String.format("values at index %s is greater than listSize %s", idx(), listSize));
```

Comment:
please revert this and use the same language above.

Reply:
ok, fixed.